packages/@react-aria/tooltip/src/useTooltip.ts

```diff
@@ -24,11 +26,16 @@ interface TooltipAria {
 /**
  * Provides the accessibility implementation for a Tooltip component.
  */
-export function useTooltip(props: AriaTooltipProps): TooltipAria {
+export function useTooltip(props: AriaTooltipProps, state?: TooltipTriggerState): TooltipAria {
```

Comment:
I've chosen to accept `TooltipTriggerState` here because it contains everything that we need here. I couldn't make it required right now as it would be a breaking change but I believe that this should be done in the future.

Reply:
This is fine I think. Maybe we should add a `console.warn` here in case `state` isn't passed? That way it's not technically breaking but people know to update their code.

Reply:
Yeah - this could be done. I was wondering - any particular reason why you have not been using `process.env.NODE_ENV` checks in the codebase so far? Such a warning would be a great candidate for a dev-only branch of code.